src/ipld/frozen_cids.rs

```diff
@@ -1,12 +1,12 @@
 // Copyright 2019-2023 ChainSafe Systems
 // SPDX-License-Identifier: Apache-2.0, MIT
 
-use crate::utils::cid::CidVariant;
+use crate::utils::cid::SmallCid;
 use cid::Cid;
 use serde::{Deserialize, Serialize};
 
 /// A wrapper around `Box<[CidVariant]>` with a [`Cid`]-friendly API:
```

Comment:
Did you intend to change the `CidVariant` here as well?

Reply:
Yes, thanks, good catch.